1-exercises/J-string-substring/exercise2.js

```diff
@@ -14,11 +14,11 @@ let names = [
   "Arron Graham",
 ];
 
-names[0] = names[0].substring();
-names[1] = names[1].substring();
-names[2] = names[2].substring();
-names[3] = names[3].substring();
-names[4] = names[4].substring();
+names[0] = names[0].substring(0,6);
+names[1] = names[1].substring(0,7);
+names[2] = names[2].substring(0,4);
+names[3] = names[3].substring(0,4);
+names[4] = names[4].substring(0,5);
```

Comment:
This does return us the correct answer to pass the test. However, the arguments in the substring are hardcoded which means that if the test conditions change the logic, unfortunately, will break.  We could use indexOf() and split() here to make our logic robust, so, it's consistent with its output no matter what the test conditions are. By the way, I fell for the same mistake if it's any consolation : )